CMakeLists.txt

```diff
@@ -639,13 +642,15 @@ endif()
 
 SET(REVO_FW_ZIP_NAME "E3D_REVO_FW_MK3_MK3S_MK3S+_${FN_VERSION_SUFFIX}.zip")
 
-add_custom_command(TARGET ALL_MULTILANG
-  POST_BUILD
-  WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/release
-  BYPRODUCTS ${CMAKE_BINARY_DIR}/release/${REVO_FW_ZIP_NAME}
-  COMMAND ${CMAKE_COMMAND} -E tar "cfv" "${REVO_FW_ZIP_NAME}" --format=zip *E3DREVO*.hex
-  COMMAND ${CMAKE_COMMAND} -E rm *E3DREVO*.hex
-  )
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
+  add_custom_command(TARGET ALL_MULTILANG
+    POST_BUILD
+    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/release
+    BYPRODUCTS ${CMAKE_BINARY_DIR}/release/${REVO_FW_ZIP_NAME}
+    COMMAND ${CMAKE_COMMAND} -E tar "cfv" "${REVO_FW_ZIP_NAME}" --format=zip *E3DREVO*.hex
+    COMMAND ${CMAKE_COMMAND} -E rm *E3DREVO*.hex
```

Comment:
Is this `rm` call needed? It is my understanding anything in `BYPRODUCTS` will be removed on `ninja clean` or when the cmake cleans up the output. 

Proposal: These hex files should be in `BYPRODUCTS` somewhere so they are automatically cleaned up. 

Probably doesnt matter :) Just an observation

Reply:
@gudnimg Frankly it doesn't cleanup the `*E3DREVO*.hex` files after compressing these without the `rm`
```
Prusa-Firmware/MK3/build/release$ ls
E3D_REVO_FW_MK3_MK3S_MK3S+_FW_3.14.0-BETA1.zip
MK25-RAMBo10a_FW_3.14.0-BETA1.zip
MK25-RAMBo13a_FW_3.14.0-BETA1.zip
MK25S-RAMBo10a_FW_3.14.0-BETA1.zip
MK25S-RAMBo13a_FW_3.14.0-BETA1.zip
MK3-E3DREVO_FW_3.14.0-BETA1_MULTILANG.hex
MK3-E3DREVO_HF_60W_FW_3.14.0-BETA1_MULTILANG.hex
MK3_FW_3.14.0-BETA1_MULTILANG.hex
MK3S_MK3S+-E3DREVO_FW_3.14.0-BETA1_MULTILANG.hex
MK3S_MK3S+-E3DREVO_HF_60W_FW_3.14.0-BETA1_MULTILANG.hex
MK3S_MK3S+_FW_3.14.0-BETA1_MULTILANG.hex
``